### DISCUSSION TITLE
Issue with AWS S3

### DISCUSSION THREAD

**Discussion by hikertommy** [osiset/laravel-shopify#658] [Q&A]
Hi all.

We have an issue with with trying to use this package with AWS s3.

When we install the any AWS package it couldn't install because of some requirements are different in laravel-shopify package. We have installed it forcefully than the routes like (/login, billing processing etc.) are stopped working.

Anyone experienced this? It’s just so we can allow file uploads.

Any help appreciated.

**Comment by mferrario** [comment]
@hikertommy ,

I have been using this package with no problems on AWS (10.2). However, I am using AWS Beanstalk to host this package/Laravel and only using S3 for uploaded files and other assets (Images, CSS, etc..). I'm not sure if you can serve a site directly from hosted S3 files... I suspect not. 

m
